seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -86,7 +88,8 @@ def test_anvil_workspace_page(self, mock_logger):
     @mock.patch('seqr.views.apis.anvil_workspace_api.add_service_account')
     @mock.patch('seqr.utils.communication_utils.EmailMultiAlternatives')
     @mock.patch('seqr.views.apis.anvil_workspace_api.does_file_exist')
-    def test_create_project_from_workspace(self, mock_file_exist, mock_email, mock_add_service_account,
+    @mock.patch('seqr.views.apis.anvil_workspace_api.file_iter')
```

Comment:
you need to add tests that `mock_file_iter` is called with the expected arguments, same as how we currenlty check `mock_file_exist`. We want at least one call with a gzipped file and at least one with a non-gzipped file